### PR TITLE
Adding extendedlocation sdk and tests

### PR DESCRIPTION
**Pull request by aditimalladi**
Adding new SDK generated from Swagger File already merged into https://github.com/Azure/azure-rest-api-specs
PR for Swagger merge : https://github.com/Azure/azure-rest-api-specs/pull/13101

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [x] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [x] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

**Comment by aditimalladi** [comment]
@m-nash Requesting review for new service under /sdk

File: sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/src/Microsoft.Azure.Management.ExtendedLocation.csproj [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ExtendedLocation</PackageId>
+    <Description>Provides Microsoft Azure ExtendedLocation operations.</Description>
+    <Version>1.0.0</Version>
```
Comment (markcowl) [inline sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/src/Microsoft.Azure.Management.ExtendedLocation.csproj:9]: You should start out as a prerelease project, I think, so version should be '1.0.0-beta.1'

File: sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <PropertyGroup>
+    <PackageId>ExtendedLocation.Tests</PackageId>
+    <AssemblyName>ExtendedLocation.Tests</AssemblyName>
+    <Version>1.0.0</Version>
+    <Description>Test Project for Automation tests</Description>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1"/>
+    <PackageReference Include="MSTest.TestAdapter" Version="2.1.1"/>
```
Comment (markcowl) [inline sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj:12]: You shouldn't use MSTest in these tests - you should use xunit

Comment (markcowl) [reply sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj:12]: we should remove these

File: sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.1"/>
+    <PackageReference Include="coverlet.collector" Version="1.3.0"/>
```
Comment (markcowl) [inline sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj:14]: I haven't seen this being used, it looks like it uses the MIT license.  hat is this used for, and is it required for these tests?

**Review by markcowl — CHANGES_REQUESTED** [review]
TestOperationsCustomLocation.cs is too large to review, can you please split this up into individual test classes?

Also, looks like the json recordings for these tests are missing?

File: sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/src/Microsoft.Azure.Management.ExtendedLocation.csproj (hunk 1 above)
Comment (aditimalladi) [reply sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/src/Microsoft.Azure.Management.ExtendedLocation.csproj:9]: What would classify a Project as a prelease project?

File: sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj (hunk 2 above)
Comment (aditimalladi) [reply sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj]: I have updated to not include these package references

File: sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj (hunk 3 above)
Comment (aditimalladi) [reply sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj]: I have updated to not include these package references

**Review by markcowl — APPROVED** [review]

**Review by LEBATRUNG — COMMENTED** [review]
****

**Pull request merged**